content/docs/integrations.mdx

```diff
@@ -109,6 +109,20 @@ If a job may not complete within the `min delay` period, it would be interrupted
 
 :::
 
-### **Client TLS Key**
+### Client TLS Key
 
 For data sources using mTLS, you can select a [client certificate](/docs/concepts/certificates) (added under **Manage** → **Certificates**) to provide to the data source.
+
+## IP range lookup support
+
+Pomerium supports Classless Inter-Domain Routing (CIDR) indexing to improve database querying performance. This is useful if you're importing a list of records containing IP ranges.
```

Comment:
The thing I was surprised about was the ability to match on IP address ranges at all (setting aside any questions to do with performance).

I'd like to see some explanation along these lines:

"When using `request.ip` as a key, you can also match against IP address ranges (expressed in CIDR notation). To do so, add a special `$index` key to your external data source records, like so:"

It also might make sense to move this up, next to the "Foreign Key" heading?